client/src/modules/reports/generate/stock_consumption_graph_report/stock_consumption_graph_report.html

```diff
@@ -0,0 +1,64 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 translate>REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          
+          <!-- select depot -->
+          <bh-depot-select
+            depot-uuid="ReportConfigCtrl.depot.uuid"
+            on-select-callback="ReportConfigCtrl.onSelectDepot(depot)"
+            required="true">
+            <bh-clear on-clear="ReportConfigCtrl.clear('depot_uuid')"></bh-clear>
+          </bh-depot-select>
+
+          <!-- select inventory -->
+          <bh-inventory-select
+            inventory-uuid="ReportConfigCtrl.inventory.uuid"
+            on-select-callback="ReportConfigCtrl.onSelectInventory(inventory)"
+            required="true"
+            only-consumable="true">
+            <bh-clear on-clear="ReportConfigCtrl.clear('inventory_uuid')"></bh-clear>
+          </bh-inventory-select>
+
+
+          <!-- date interval -->
+          <bh-date-interval
```

Comment:
Let's make this select an entire period instead of between two dates.  That will make the logic a lot easier to maintain and ensure that we always have the same size data.

Also, since most of the stock indicators are calculated based on monthly indicators, it makes sense in that respect.